cfg/common_test.go

```diff
@@ -187,8 +187,8 @@ backendsByCluster:
       - "http://10.190.202.32:8080"
       - "http://10.190.197.92:8080"
       protocolBackends:
-      - address: "http://10.190.198.92:7070"
-        grpc: true
+      - http: "http://10.190.198.93:8080"
+        grpc: "10.190.198.93:7070"
```

Comment:
I guess we would need to note that it's not backward compatible in the pr due to the config changes.

Reply:
Actually it is. the older `backends` still works. This new list is called `protocolBackends`.

Reply:
oh, I see. you know what, we actually go with 'grpc://10.190.198.93:7070' and skip a new config field. but just a wild idea.

Reply:
Well I thought about this but the issue is that we should have a fall back strategy for find, info, etc. as they are not yet implemented in gRPC. So with the current changes, Renders hit the gRPC endpoint and others use the older implementation. Although, I am open to change the plan to implement all of these rpcs prior to going on production but that will require more changes, hence, more time and risk.